systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -181,16 +181,19 @@ protected void installClusterOperator(ExtensionContext extensionContext, String
     }
 
     public void installClusterWideClusterOperator(ExtensionContext extensionContext, String namespace, long operationTimeout, long reconciliationInterval) {
-        prepareEnvForOperator(extensionContext, namespace);
-        // Apply role bindings in CO namespace
-        applyBindings(extensionContext, namespace);
-
-        // Create ClusterRoleBindings that grant cluster-wide access to all OpenShift projects
-        List<ClusterRoleBinding> clusterRoleBindingList = ClusterRoleBindingTemplates.clusterRoleBindingsForAllNamespaces(namespace);
-        clusterRoleBindingList.forEach(clusterRoleBinding ->
-            ClusterRoleBindingResource.clusterRoleBinding(extensionContext, clusterRoleBinding));
-        // 060-Deployment
-        resourceManager.createResource(extensionContext, BundleResource.clusterOperator(namespace, "*", operationTimeout, reconciliationInterval).build());
+        // deploy cluster operator only once for parallel STs
+        if (StUtils.isClusterOperatorAlreadyDeployed(namespace)) {
```

Comment:
We deploy CO to some testing namespace, right? Shouldn't we deploy it to some `infra` namespace always? It might make some things easier like finding where CO is installed.

Reply:
Yes, I agree....will do it like that.

Reply:
I have changed it and this kind of stuff will do afterwards...It quite complex to do and not related to this PR it as we discussed offline